decidim-core/lib/decidim/content_block_manifest.rb

```diff
@@ -45,6 +46,12 @@ def cell(cell_name)
       self.cell_name = cell_name
     end
 
+    # Public: Registers the I18n key this contnt block will use to retrieve its
+    # public name. Use `#i18n_name_key` to retrieve it.
+    def public_name_key(i18n_key)
```

Comment:
Why don't use `i18n_name_key` directly?

Reply:
Because of the API to register the fields. If I use it directly, then we should use `component_block.i18n_name_key = "foo.bar"` on the registration step, but this API is different from the one to add images and options (`content_block.image :my_image`), and I wanted all fields to be used the same way.

This strategy is also used for the `cell`/`cell_name` part.